src/Transliterate.jl

```diff
@@ -21,24 +25,22 @@ julia> transliterate("≠ ∉"; custom_replacements=Dict("≠" => "not equal", "
 "not equal not in"
 ```
 """
-function transliterate(str; languages=[], custom_replacements=Dict())
-    if typeof(languages) <: AbstractString
-        languages = [languages]
-    end
-    if languages == []
-        languages = collect(keys(replacements))
-    end
+transliterate(str; languages = nothing, custom_replacements = Dict()) = transliterate(str, wrap_languages(languages); custom_replacements = custom_replacements)
+
+function transliterate(str, languages; custom_replacements = Dict())
+    str = foldl(replace, custom_replacements, init = str)
+
     if "la" ∉ languages
-        pushfirst!(languages, "la")
+        str = foldl(replace, replacements[1].second, init = str)
     end
 
     for language in languages
-        merge!(custom_replacements, replacements[language])
+        # TODO: Here should be some warnings/safety belts added
```

Comment:
Which warnings exactly?

Reply:
If user provides wrong language (which doesn't exist in `replacements`) this function will fail with rather confusing message for user. Some policy should be adopted in case of wrong inputs: either silently ignore them, or raise error, or issue warnings and ignore.